views/help_menu/mentorship_message.txt.erb

```diff
@@ -0,0 +1,2 @@
+TODO TEXT GOES HERE
```

Comment:
Lets add the following text here.

The Operation Code Mentorship program aims to pair you with an experienced developer in order to further your programming or career goals. When you sign up for our mentorship program you'll fill out a form with your experience and interests. You'll then be paired up with an available mentor that best meets those interests.

If you're interested in joining our mentorship program please fill out our sign up form here: http://op.co.de/mentee-enrollment.